website/cue/reference/components/sinks/http.cue

```diff
@@ -39,11 +39,6 @@ components: sinks: http: {
 			proxy: enabled: true
 			request: {
 				enabled:                    true
-				concurrency:                10
-				rate_limit_duration_secs:   1
-				rate_limit_num:             1000
-				retry_initial_backoff_secs: 1
-				retry_max_duration_secs:    10
 				timeout_secs:               30
```

Comment:
Same here, I'd be game to just make this the default of 60.

Reply:
Resolved in f2734ac